(REGRESSION 275453@main) Broke clean cmake builds

lauromoura opened the pull request:
#### 73271df67cecac670d2b19afefec29d0b6c20e1b
<pre>
(REGRESSION 275453@main) Broke clean cmake builds
<a href="https://bugs.webkit.org/show_bug.cgi?id=270239">https://bugs.webkit.org/show_bug.cgi?id=270239</a>

Reviewed by Don Olmstead.

Fix entry for file that was moved to cocoa-specific directory.

* Source/WebKit/UIProcess/XR/PlatformXRSystem.h: Drive-by non-unified
  fix that was breaking unified WPE build on flatpak
* Source/WebKit/Sources.txt:
* Source/WebKit/SourcesCocoa.txt:

Canonical link: <a href="https://commits.webkit.org/275480@main">https://commits.webkit.org/275480@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/788b08c4ff6299154112bdaec6210734b84d3aa6

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/41955 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/20973 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/44348 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/44541 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/38053 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/24139 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/18301 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/34770 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/42529 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/17900 "Passed tests") | [  ~~🧪 api-mac~~](https://ews-build.webkit.org/#/builders/18/builds/36131 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 api-wpe~~](https://ews-build.webkit.org/#/builders/41/builds/15703 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [  ~~🧪 ios-wk2-wpt~~](https://ews-build.webkit.org/#/builders/42/builds/15580 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/45952 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/38144 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/37489 "Passed tests") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/41401 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/16771 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/50/builds/13779 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 api-gtk~~](https://ews-build.webkit.org/#/builders/21/builds/39830 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/18390 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/9397 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/18449 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/18035 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

donny-dont commented on Source/WebKit/Sources.txt:
```diff
@@ -880,7 +880,6 @@ IPCTesterMessageReceiver.cpp
 IPCTesterReceiverMessageReceiver.cpp
 ModelProcessModelPlayerManagerProxyMessageReceiver.cpp
 ModelProcessModelPlayerMessageReceiver.cpp
-ModelProcessModelPlayerProxyMessageReceiver.cpp
```
Move this into `SourcesCocoa.txt`

lauromoura replied:
Nice catch. Thanks!

lauromoura committed:
(REGRESSION 275453@main) Broke clean cmake builds
https://bugs.webkit.org/show_bug.cgi?id=270239

Reviewed by Don Olmstead.

Fix entry for file that was moved to cocoa-specific directory.

* Source/WebKit/UIProcess/XR/PlatformXRSystem.h: Drive-by non-unified
  fix that was breaking unified WPE build on flatpak
* Source/WebKit/Sources.txt:
* Source/WebKit/SourcesCocoa.txt:

Canonical link: https://commits.webkit.org/275480@main
